doc/classes/Callable.xml

```diff
@@ -4,7 +4,7 @@
 		A built-in type representing a method or a standalone function.
 	</brief_description>
 	<description>
-		[Callable] is a built-in [Variant] type that represents a function. It can either be a method within an [Object] instance, or a standalone function not related to any object, like a lambda function. Like all [Variant] types, it can be stored in variables and passed to other functions. It is most commonly used for signal callbacks.
+		[Callable] is a built-in [Variant] type that represents a function. It can either be a method within an [Object] instance, or an independent function not related to any object, like a lambda function. Like all [Variant] types, it can be stored in variables and passed to other functions. It is most commonly used for signal callbacks.
```

Comment:
```suggestion
		[Callable] is a built-in [Variant] type that represents a function. It can either be a method within an [Object] instance, or an independent function not related to any object, like a lambda function. Like all [Variant] types, it can be stored in variables and passed to other functions. It is most commonly used for signal callbacks.
```

An independent function doesn't sound like anything to me. IMO "standalone" is a valid term for this problem space and should not be avoided. Although I agree that in the example of "STATE_MACHINE_TYPE_GROUPED" its used out of place.